otio_openassetio/operations/openassetio_media_linker.py

```diff
@@ -0,0 +1,175 @@
+# Copyright The Foundry Visionmongers Ltd
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+"""
+A media linker that resolves media reference URLs via OpenAssetIO if
+their target_url is set to a valid entity reference.
+"""
+
+from collections import namedtuple
+
+from openassetio import log
+from openassetio.hostAPI import HostInterface, Session
+from openassetio.specifications import LocaleSpecification
+from openassetio.pluginSystem import PluginSystemManagerFactory
+
+import opentimelineio as otio
+
+
+#
+# Media Linker
+#
+
+
+def link_media_reference(in_clip, media_linker_argument_map):
+
+    # Once https://github.com/TheFoundryVisionmongers/OpenAssetIO/issues/247
+    # is implemented, we can more concisely handle ImageSequenceReference,
+    # but we will need to convert file:///a/path/sequence.%04d.ext to the
+    # component fields required by OTIO. For now, we only handle simple
+    # ExternalReference schemas.
+    # We can then also set all other relevant properties of the
+    # Clip/MediaReference based on trait properties.
+    # For now, keep it simple and just resolve the URL
+
+    mr = in_clip.media_reference
+
+    if not mr or not isinstance(mr, otio.schema.ExternalReference):
```

Comment:
[consider][minor] Redundant `not mr or`.

Reply:
Removed [here](https://github.com/TheFoundryVisionmongers/otio-openassetio/pull/1/commits/293d6914e31181dabcd2f2125c84f85cd2e221b5).